src/component/Label.tsx

```diff
@@ -54,20 +54,22 @@ type PolarLabelPosition = 'insideStart' | 'insideEnd' | 'end';
 
 export type LabelPosition = CartesianLabelPosition | PolarLabelPosition;
 
+export type LabelFormatter = (label: RenderableText) => RenderableText;
+
 interface LabelProps {
   viewBox?: ViewBox;
   parentViewBox?: ViewBox;
-  formatter?: (label: React.ReactNode) => React.ReactNode;
```

Comment:
this is a breaking change though technically 😓. Not because of what renders or not, but because someone in userland probably has something like `<>{someText}</>` because we accepted ReactNode